pinot-core/src/main/java/com/linkedin/pinot/core/segment/creator/InvertedIndexCreator.java

```diff
@@ -40,34 +40,17 @@
 public interface InvertedIndexCreator extends Closeable {
 
   /**
-   * Add an entry for single-value column.
-   *
-   * @param docId Document id
-   * @param dictId Dictionary id
+   * For single-valued column, adds the dictionary Id for the next document.
    */
-  void addSV(int docId, int dictId);
+  void add(int dictId);
```

Comment:
Can you please add a comment that all implementations of this interface must be  stateful, and you can only add items in sequence of docId? thanks

Reply:
Added more javaDoc for the interface